modules/plugin/shapefile/src/main/java/org/geotools/data/shapefile/ShapefileDataStoreFactory.java

```diff
@@ -82,7 +82,7 @@
      */
     public static final Param FILE_TYPE = new Param("filetype", String.class,
             "Discriminator for directory stores", false, "shapefile", new KVP(Param.LEVEL,
-                    "advanced"));
+                    "program"));
```

Comment:
yes, sorry I added program later when it became obvious some things were really only machine editable